eclair-core/src/test/scala/fr/acinq/eclair/channel/states/h/ClosingStateSpec.scala

```diff
@@ -89,15 +89,19 @@ class ClosingStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with
         bob2blockchain.expectMsgType[WatchFundingConfirmed]
         awaitCond(alice.stateName == WAIT_FOR_FUNDING_CONFIRMED)
         awaitCond(bob.stateName == WAIT_FOR_FUNDING_CONFIRMED)
-        system.eventStream.subscribe(eventListener.ref, classOf[TransactionPublished])
-        system.eventStream.subscribe(eventListener.ref, classOf[TransactionConfirmed])
+        alice.underlying.system.eventStream.subscribe(eventListener.ref, classOf[TransactionPublished])
+        alice.underlying.system.eventStream.subscribe(eventListener.ref, classOf[TransactionConfirmed])
+        bob.underlying.system.eventStream.subscribe(eventListener.ref, classOf[TransactionPublished])
+        bob.underlying.system.eventStream.subscribe(eventListener.ref, classOf[TransactionConfirmed])
```

Comment:
It's probably worth going the extra mile here as well and have one `eventListener` per peer, our assertions will be more accurate.

Reply:
I did tinker with that, and backed off when I saw random tests failing and a new rabbit hole opening. We definitely need to rework that.

Reply:
Let's keep that for a separate PR then!